### PR TITLE
fix(PopperContent): Use create portal instead of unstable_renderSubtr…

### PR DESCRIPTION
**Pull request by Imundy**
…eeIntoContainer

This update *should* have no visible effects.

- [x] Bug fix  
- [ ] New feature 
- [ ] Chore 
- [ ] Breaking change 
- [ ] There is an open issue which this change addresses
- [x] I have read the **[CONTRIBUTING](./CONTRIBUTING.md)** document.
- [x] My commits follow the [Git Commit Guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- - [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

I am getting an obscure error occasionally with `unstable_renderSubtreeIntoContainer` on react versions newer than 15.4.1 (possibly older versions too) where react *thinks* the container is not mounted. It is the same error as https://github.com/reactstrap/reactstrap/issues/1216
I unfortunately am having trouble creating a minimally reproducible example 😕

### CODE REVIEW

File: src/__tests__/Tooltip.spec.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ describe('Tooltip', () => {
 
     const tooltips = document.getElementsByClassName('tooltip');
 
-    expect(wrapper.find('.tooltip.show').hostNodes().length).toBe(0);
+    expect(wrapper.find('.tooltip.show').hostNodes().length).toBe(1);
```
Comment (Imundy) [inline src/__tests__/Tooltip.spec.js]: I *think* this is actually the desired behavior. before the PopperContent was creating a div manually and then mounting into that div. Now it's wrapping the content in the div while it creates the portal which means we do have a host node, but the dom ends up being the same as before.

**Comment by kyriiherman** [comment]
@Imundy actually, there is an open issue (but I'm not sure) https://github.com/reactstrap/reactstrap/issues/1201

**Comment by EugeneChu** [comment]
Tested the version of `reactstrap` pointed to this commit in our project. We still have problems with `PopperContent` and passing elements with `Context` as it's children

**Comment by eugenechu** [comment]
⬆️ is so much needed, @TheSharpieOne we would so much appreciate a minor release with this thing!
Thanks in advance

**Comment by verbold** [comment]
+1 on a minor release. This would allow reactstrap to be compatible with the [v6.0.0](https://github.com/reduxjs/react-redux/releases) react-redux release.

**Pull request merged**